client-app/src/i18n.ts

```diff
@@ -0,0 +1,17 @@
+import Vue from "vue";
+import VueI18n from "vue-i18n";
+import LocalizationService from "@common/localization.service";
+
+Vue.use(VueI18n);
+
+const i18n = new VueI18n({
+  locale: process.env.VUE_APP_I18N_LOCALE || "en",
+  fallbackLocale: process.env.VUE_APP_I18N_FALLBACK_LOCALE || "en",
+  messages: {}
+});
+
+LocalizationService.get().then(({ data }) => {
```

Comment:
I don't remember exactly why but  `i18n.setLocaleMessage("en", data);` didn't work before in this place. All your resources were untranslated.

Reply:
Will check

Reply:
@tatarincev Working perfectly:
![image](https://user-images.githubusercontent.com/6369252/73835625-875e7a00-482f-11ea-93eb-1a4aea77b790.png)
![image](https://user-images.githubusercontent.com/6369252/73835635-8c232e00-482f-11ea-95e2-5eb30b39bf35.png)